frontend/src/LoginDialogue.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import styles from './css/LoginDialogue.module.css';
+
+export default function FormDialog() {
+  const [open, setOpen] = React.useState(false);
```

Comment:
you can just do a "static" import by doing `import React, { useState } from 'react'`;

Reply:
This way you don't have to do `React.useState`